oak-search/src/main/java/org/apache/jackrabbit/oak/plugins/index/search/util/ConfigUtil.java

```diff
@@ -36,34 +36,71 @@
  */
 public class ConfigUtil {
 
+    private static final String ILLEGAL_STATE_EXCEPTION_ERROR_MESSAGE = "Multiple Values provided for property %s in Index Defintion . Single value was expected";
+
     public static boolean getOptionalValue(NodeState definition, String propName, boolean defaultVal){
-        PropertyState ps = definition.getProperty(propName);
-        return ps == null ? defaultVal : ps.getValue(Type.BOOLEAN);
+        try{
+            PropertyState ps = definition.getProperty(propName);
+            return ps == null ? defaultVal : ps.getValue(Type.BOOLEAN);
+        }catch(IllegalStateException e ){
```

Comment:
Please add spaces after "}", before "{", around "()", and remove the superfluous space after e.

Reply:
What I usually do is use the IDE formatter, but only for the selected text. (Eclipse allows this, and I'm almost sure IntelliJ as well.)